src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -143,7 +141,30 @@ def _expand_config_op_types(self, config):
                 expanded_op_types.append(op_type)
         return expanded_op_types
 
-
+class PrunerLayerWrapper(torch.nn.Module):
+    def __init__(self, module, name, type, config, pruner):
+        super().__init__()
+        # origin layer information
+        self.module = module
+        self.name = name
+        self.type = type
+        # config and pruner
```

Comment:
It seems `self.type` and `self.config` are not used in the class.

Reply:
these parameters could be used in pruner.